CHANGELOG/CHANGELOG-0.9.md

```diff
@@ -1,5 +1,19 @@
+## v0.9.3
+
+Changes since `v0.9.2`:
+
 ## v0.9.2
 
+Changes since `v0.9.1`:
+
+## Changes by Kind
```

Comment:
This should be put up in the v0.9.3 section

Reply:
Yes, that's right